spec/models/endpoint_spec.rb

```diff
@@ -175,15 +161,10 @@
       expect(Endpoint.which_need_archive_copy(druid, version - 1).pluck(:endpoint_name)).to eq %w[mock_archive1]
     end
 
-    it 'does not allow SQL injection' do
-      # a bit too whitebox, but oh well...  specifically testing version field
-      # because we know we're not using the usual AR params for bind vars approach
-      # in the subquery.  we cast version.to_i, which will silently convert anything
-      # it can't figure out how to properly translate into zero.
-      bogus_version = "0)))); GRANT ALL PRIVILEGES ON DATABASE pres TO icanhaznjctionattk; DROP TABLE students; --"
-      generated_sql = Endpoint.which_need_archive_copy(druid, bogus_version).to_sql
-      expect(generated_sql).not_to match(/#{Regexp.escape(bogus_version)}/)
-      expect(generated_sql).to match(/"preserved_copies"."version" = 0/)
+    it 'Casts version to integer' do
```

Comment:
I would like to at least retain a comment about why we care (avoid SQL injection b/c if the version is being cast to an integer, it isn't possible to deliver any valuable SQL payload via it) in the code itself.